### ISSUE TITLE
FortiOS password issue (hash changed, secrets not removed)

### ISSUE THREAD

**Issue by mediumo** [ytti/oxidized#697]
FortiOS 5.4 seems to change the encrypted passwords everytime it does a "show" command

any chance of ignoring this in the config-diff to not store a new config?`
But it would be great to store encrypted passwords nevertheless since the private-keys for internal certificate generation are stored with "set password ENC", too.

additional patch for the model, add secret removal (can bypass the problem aboce:

```
  cmd :secret do |cfg|
    cfg.gsub! /(set password).*/, '\\1 <configuration removed>'
    cfg.gsub! /(set passwd).*/, '\\1 <configuration removed>'
    cfg
  end

```

If OK I can submit a pull request

**Comment by mediumo** [comment]
secret removal fixed in #698

**Comment by ytti** [comment]
Thanks, if you had added in commit messages ```fixes #697``` or ```closes #697```this ticket would have been automatically closed once merged.
